test/wh_test_she.c

```diff
@@ -59,6 +60,23 @@ enum {
         BUFFER_SIZE = 4096,
     };
 
+/* Helper function to destroy a SHE key so the unit tests don't
+ * leak NVM objects across invocations. Necessary, as SHE doesn't expose a
+ * destroy key API since SHE keys are supposed to be fixed hardware keys */
+static int _destroySheKey(whClientContext* client, whNvmId clientSheKeyId)
```

Comment:
Hrmmm.  This could be added to our SHE library, even if it is outside of the SHE spec.